src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -11,7 +11,7 @@ import classNames from 'classnames';
 import ActiveState from './ActiveState';
 import ApprovalCommentForm from '../approval-comment-form';
 import EmptyState from './EmptyState';
-import { collapseFeedState, shouldShowEmptyState } from './activityFeedUtils';
+import { collapseFeedState, shouldShowEmptyState, uuidv4 } from './activityFeedUtils';
```

Comment:
Is this UUID being persisted?
If this is only client side, and you just want an ID temporarily, use lodash/uniqueid